scene/resources/font.cpp

```diff
@@ -257,8 +257,8 @@ Error BitmapFont::create_from_fnt(const String &p_file) {
 
 			if (keys.has("file")) {
 
-				String file = keys["file"];
-				file = p_file.get_base_dir() + "/" + file;
+				String base_dir = p_file.get_base_dir();
+				String file = base_dir.plus_file(keys["file"]);
```

Comment:
That looks cleaner, but it doesn't seem to change the logic at all, so how does it fix a bug?

Reply:
String's plus_file function only adds a slash between the two parts if they're non-empty and don't already have one.
Previously, if the fnt file was in the project's root directory an extra slash would be added and the texture wouldn't be loaded

Reply:
It should probably be fixed so that `res:///` is reduced to `res://`, but with this fix it's cleaner anyway (and for such a change to the file loading logic, better wait for 3.1 :))